src/lib/core/fiber.h

```diff
@@ -170,6 +170,10 @@ enum {
 	 * This flag is set when fiber has custom max slice.
 	 */
 	FIBER_CUSTOM_SLICE	= 1 << 8,
+	/**
+	 * This flag idicates, if the fiber can be killed from the Lua world.
+	 */
+	FIBER_IS_SYSTEM         = 1 << 9,
```

Comment:
Just to bring some attention of people who are working on it: I placed some thoughts around this approach to fix cartridge's weird hot reload into the issue: https://github.com/tarantool/tarantool/issues/7473#issuecomment-1203235216.

Reply:
And... whether anybody consider dropping the code that cancels fibers from cartridge itself? Why we can't?